index.html

```diff
@@ -155,7 +155,7 @@ <h3>Skills</h3>
     </main>
     <footer>
       <form
```

Comment:
- [ ] You did a good job with the project functionalities but there is a little issue, If the validation is OK, the form should be submitted. You are missing an action attribute with formspree URL from your form, please make sure to add that when the validation is ok, the form should be successfully submitted.